model/release.go

```diff
@@ -146,11 +147,57 @@ func (tag *Tag) UnmarshalJSON(b []byte) error {
 	return nil
 }
 
+type Notes string
+
+var (
+	NotesLengthMaximumCharacters = 1024
+
+	ErrReleaseNotesTooLong  = errors.New("release notes too long")
+	ErrCharactersNotAllowed = errors.New("release notes contain characters which are not allowed")
+)
+
+type InvalidCharError struct {
+	Offset int
+	Char   byte
+}
+
+func (err InvalidCharError) Error() string {
+	return fmt.Sprintf(`invalid character '%c' at offset %d`, err.Char, err.Offset)
+}
+
+func IsNotGraphic(r rune) bool {
+	return !unicode.IsGraphic(r)
+}
+
+func (n Notes) Validate() error {
+	length := len(n)
+	if length > NotesLengthMaximumCharacters {
+		return ErrReleaseNotesTooLong
+	}
+	if i := strings.IndexFunc(string(n), IsNotGraphic); i > 0 {
+		return InvalidCharError{
```

Comment:
```suggestion
		return &InvalidCharError{
```

Reply:
no problema (I went the other way and removed the * from the receiver)